imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,98 @@
+"""."""
+
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.encoding import python_2_unicode_compatible
+
+CAMERA_TYPE = (
+    ("iphone", "iphone"),
+    ("SamSung", "SamSung"),
+    ("Nikon", "Nikon"),
+    ("Canon", "Canon")
+)
+
+PHOTOGRAPHY_STYLE = (
+    ("Color", "Color"),
+    ("Black and White", "Black and White"),
+    ("Portrait", "Portrait"),
+    ("Landscape", "Landscape")
+)
+
+SOCIAL_STATUS = (
+    ('Peasant', 'Peasant'),
+    ('Royalty', "Royalty"),
+    ('Merchant', 'Merchant'),
+    ('Scholar', 'Scholar')
+)
+
+AGE_CHOICES = [(i, i) for i in range(0, 100)]
+
+
+class ProfileManager(models.Manager):
+    """Manage profiles."""
+
+    def get_queryset(self):
+        """Return all active users."""
+        return super(ProfileManager, self).get_queryset().filter(user__is_active=True).all()
+
+
+# Create your models here.
+class ImagerProfile(models.Model):
+    """A profile for users to our applications."""
+
+    user = models.OneToOneField(
+        User,
+        related_name="profile",
+        on_delete=models.CASCADE
+    )
+    camera_type = models.CharField(
+        max_length=25,
+        choices=CAMERA_TYPE,
+        default="iphone",
+        null=True
+    )
+    photography_style = models.CharField(
+        max_length=25,
+        choices=PHOTOGRAPHY_STYLE,
+        default="color",
+        null=True
```

Comment:
`null = True` is typically used for integer fields, for string we would use `blank=True`